targets/xbmc

```diff
@@ -1,35 +0,0 @@
-#!/bin/sh -e
```

Comment:
I've created the minimal target, but i'm unsure what exactly to add to installer/main.sh.

I assume it would be around line 306, but i'm not familiar with the syntax.